### ISSUE TITLE
revise internal function documentation

### ISSUE THREAD

**Issue by twhiteaker** [jpshanno/ingestr#33]
I believe the docstring in internals.R is just for the `check_inputs` function. However, in the HTML page generated from it, it appears that the documentation is for all functions in that script. 
file:///C:/Users/whiteatl/Documents/ingestr/docs/reference/check_inputs.html

Could this be confusing since `check_inputs` takes a class.name argument but the other functions don't, for example?

I suggest revising the documentation to make it clearer. However, I'm not sure how. I tried adding a docstring to all_character() so it could have its own description, but pkgdown said it didn't have a name or title so it skipped it.

**Comment by jpshanno** [comment]
Yeah they are internal functions, but I was thinking of having them with real help pages for someone who is creating an ingestr function. You're right about the confusion, I think that `check_inputs()` should be hidden, and the help page should just show the wrappers around `check_inputs()`
